integrationTests/vm/arwen/arwenvm/mandosConverter/mandosConverter.go

```diff
@@ -0,0 +1,150 @@
+package mandosConverter
+
+import (
+	"bytes"
+	"errors"
+	"math/big"
+
+	mge "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/mandos-go/elrondgo-exporter"
+	mgutil "github.com/ElrondNetwork/arwen-wasm-vm/v1_4/mandos-go/util"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/ElrondNetwork/elrond-go/state"
+	"github.com/ElrondNetwork/elrond-go/testscommon/txDataBuilder"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+var errReturnCodeNotOk = errors.New("returnCode is not 0(Ok)")
+
+// CreateAccountsFromMandosAccs uses mandosAccounts to populate the AccountsAdapter
+func CreateAccountsFromMandosAccs(tc *vm.VMTestContext, mandosUserAccounts []*mge.TestAccount) (err error) {
+	for _, mandosAcc := range mandosUserAccounts {
+		acc, err := tc.Accounts.LoadAccount(mandosAcc.GetAddress())
+		if err != nil {
+			return err
+		}
+		account := acc.(state.UserAccountHandler)
+		account.IncreaseNonce(mandosAcc.GetNonce())
+		err = account.AddToBalance(mandosAcc.GetBalance())
+		if err != nil {
+			return err
+		}
+
+		mandosAccStorage := mandosAcc.GetStorage()
+		for key, value := range mandosAccStorage {
+			err = account.DataTrieTracker().SaveKeyValue([]byte(key), value)
+			if err != nil {
+				return err
+			}
+		}
+
+		accountCode := mandosAcc.GetCode()
+		if len(accountCode) != 0 {
+			account.SetCode(accountCode)
+			ownerAddress := mandosAcc.GetOwner()
+			account.SetOwnerAddress(ownerAddress)
+			account.SetCodeMetadata([]byte{0, 0})
```

Comment:
Does `mandosAcc` contain code metadata to use here instead of `0`?

Reply:
no